.github/actions/spelling/line_forbidden.patterns

```diff
@@ -19,6 +24,12 @@
 # s.b. greater than
 \bgreater then\b
 
+# s.b. into
+#\sin to\s
```

Comment:
This has a fairly high false positive rate, but https://github.com/nasa/fprime/pull/1790/files#diff-86d8cfc399b755e3ae48dffc23ef124898fe465f198f8d8777d516d3ec386238 is an example where it was right. Since it's wrong more often than not, I'm disabling it.

Reply:
Works for me